Dockerfile

```diff
@@ -0,0 +1,85 @@
+##############################################################################
+################## Meant to be run with Google Clould Build ##################
+##############################################################################
+
+# From the root of the repo, use the following command to run:
+#   gcloud builds submit --tag us-docker.pkg.dev/linera-io-dev/linera-docker-repo/<PACKAGE_NAME>:<VERSION_TAG> --timeout="3h" --machine-type=e2-highcpu-32
+# The package tag needs that prefix so it stores it in the proper Docker image repo on GCP.
```

Comment:
```suggestion
# The package name needs that prefix so it stores it in the proper Docker image repo on GCP.
```

And I'd even spell out "Google Cloud Platform", at least the first time it's mentioned.